packages/devtools_app/test/extentsion_points/extension_external_test.dart

```diff
@@ -0,0 +1,20 @@
+// Copyright 2023 The Chromium Authors. All rights reserved.
```

Comment:
nit name test so that it matches file name. Need an 's' after extension to match the file it is testing.